### PR TITLE
Add analog-style ladder filter

### PR DESCRIPTION
**Pull request by daslyfe**
adapted from https://github.com/TheBouteillacBear/webaudioworklet-wasm?tab=MIT-1-ov-file

example pattern: 
note("c f g g a c d4").fast(2)
 .sound('sawtooth')
 .lpf(200).fanchor(0)
 .lpenv(4).lpq(1).drive("<.5 0 2>")
 .ftype("ladder")

### CODE REVIEW

**Comment by felixroos** [comment]
sounds super nice! I've noticed the lpq range is different from the other filter types. I think it should be somewhat equal to avoid surprises when switching types

**Comment by daslyfe** [comment]
> sounds super nice! I've noticed the lpq range is different from the other filter types. I think it should be somewhat equal to avoid surprises when switching types

cool I updated it to be as similar as I could with the different response curves

File: packages/superdough/superdough.mjs [hunk 1]
```diff
@@ -288,24 +284,25 @@ export const superdough = async (value, t, hapDuration) => {
...
+    drive = 0.69,
     // low pass
     cutoff,
     lpenv,
     lpattack,
     lpdecay,
     lpsustain,
     lprelease,
-    resonance = 1,
+    resonance = 0.5,
```
Comment (felixroos) [inline packages/superdough/superdough.mjs:297]: is this still needed after taming the resonance curve? would be a breaking change

File: test/__snapshots__/shared.test.mjs.snap
Comment (felixroos) [inline test/__snapshots__/shared.test.mjs.snap:1]: not sure if we still need this one.. probably not

File: packages/superdough/superdough.mjs (hunk 1 above)
Comment (daslyfe) [reply packages/superdough/superdough.mjs]: updated the default back to normal :)

**Pull request merged**